core/build.gradle

```diff
@@ -305,9 +305,11 @@ task('verifyVersions') {
      * as possible after release. */
     Set<String> actualVersions = new TreeSet<>(
       indexCompatVersions
-        .findAll { false == it.unreleased }
+        .findAll { false == it.snapshot }
         .collect { it.toString() })
 
+    // TODO this is almost certainly going to fail on 5.4 when we release 5.5.0
```

Comment:
What do you mean by "fail on 5.4"? The 5.4 branch? I think we need to filter the actualVersions for things less than the current version?